tactical-microgrid-standard/CMakeLists.txt

```diff
@@ -27,7 +27,8 @@ add_library(Commands_Idl)
 opendds_target_sources(Commands_Idl
   PUBLIC
     cli_idl/CLICommands.idl
-  OPENDDS_IDL_OPTIONS -Lc++11 -I${CMAKE_SOURCE_DIR}
+  OPENDDS_IDL_OPTIONS -Lc++11 -I${CMAKE_CURRENT_SOURCE_DIR}
+  INCLUDE_BASE ${CMAKE_CURRENT_SOURCE_DIR}
```

Comment:
I also found on my local build that without this some targets failed to build because the header file path `cli_idl/***.h` does not exist.

Reply:
Yeah, it simulates the source directory structure inside the build directory.